action.yaml

```diff
@@ -10,46 +10,48 @@ branding:
   icon: "package" # icon name from Feather open source icons pack
   color: "orange"
 
-permissions:
```

Comment:
Removing permissions is **NOT** a good idea. Please add `read` permissions.

Reply:
`permissions` is not supposed to be in `action.yaml`. VS code warns me that